Doc/whatsnew/3.9.rst

```diff
@@ -439,6 +439,12 @@ Changes in the Python API
   :data:`~errno.EBADF` error.
   (Contributed by Victor Stinner in :issue:`39239`.)
 
+* :func:`codecs.lookup` now normalizes the encoding name the same way than
```

Comment:
```suggestion
* :func:`codecs.lookup` now normalizes the encoding name the same way as
```

Reply:
Oh. I copied the NEWS entry from commit 20f59fe1f7748ae899aceee4cb560e5e1f528a1f. If there is a typo, it should also be fixed in Misc/NEWS.d/next/Core and Builtins/2019-08-20-04-36-37.bpo-37751.CSFzUd.rst.

Reply:
I create a new PR in #23096, and this word have been replaced.